x-pack/plugins/security_solution/public/management/common/utils.ts

```diff
@@ -28,18 +28,18 @@ const getPolicyQuery = (policyId: string): string => {
   return `exception-list-agnostic.attributes.tags:"policy:${policyId}"`;
 };
 
-export const parsePoliciesToKQL = (includedPolicies: string, excludedPolicies: string): string => {
+export const parsePoliciesToKQL = (
+  includedPolicies: string[],
+  excludedPolicies: string[]
+): string => {
   if (isEmpty(includedPolicies) && isEmpty(excludedPolicies)) return '';
 
-  const parsedIncludedPolicies = includedPolicies ? includedPolicies.split(',') : undefined;
-  const parsedExcludedPolicies = excludedPolicies ? excludedPolicies.split(',') : undefined;
-
-  const includedPoliciesKuery = parsedIncludedPolicies
-    ? parsedIncludedPolicies.map(getPolicyQuery).join(' OR ')
+  const includedPoliciesKuery = includedPolicies
```

Comment:
nit: includedPolicies will never be undefined.

Reply:
changes in https://github.com/elastic/kibana/pull/121305/commits/d5ac895d5ad8b5969df21e3387f9345ce31ea65e